doc/manuals/admin/running.md

```diff
@@ -35,7 +35,8 @@ The configuration used by the contextBroker service is stored in the
 /etc/sysconfig/contextBroker file, which typical content is:
 
 ```
-# BROKER_USER - What user to run orion-broker as
```

Comment:
I have take the opportunity to upgrade this part of the documentation. It was a bit obsolete, based in an old version of the sysconfig file.

NTC (just informative)